Cargo.toml

```diff
@@ -16,6 +16,10 @@ name = "conformance"
 path = "src/conformance/main.rs"
 test = false
 
+[[bin]]
+name = "sh_migrate"
```

Comment:
I am wondering if this is a good name for the executable. The `sh` is ambiguous so I would prefer `softhsm_migrate` to be more clear. I think being it short is really not an issue as it should be one-off tool and not something to use on daily basis.

Reply:
ack